apps/vercel/src/connectors/users.ts

```diff
@@ -0,0 +1,47 @@
+import { env } from '@/env';
+import { VercelError } from './commons/error';
+
+
+export type VercelUser = {
+  role: string;
+  uid: string;
+  name: string;
+  email: string;
+};
+
+export type Pagination = {
+  count: number;
+  next: string | null;
+  prev: string | null;
+  hasNext: boolean;
+};
+
+type GetTeamMembersResponseData = { members: VercelUser[]; pagination: Pagination };
+
+export const getUsers = async (token: string, teamId: string, page:string|null) => {
+  const url = `https://api.vercel.com/v2/teams/${teamId}/members${page !== null ? `?limit=${env.USERS_SYNC_BATCH_SIZE}&until=${page}` : `?limit=${env.USERS_SYNC_BATCH_SIZE}`}`;
+
+  
```

Comment:
```suggestion
  const url = new URL('https://api.vercel.com/v2/teams/${teamId}/members')
  url.seachParams.append('limit', String(env.USERS_SYNC_BATCH_SIZE))
  if (page) {
    url.searchParams.append('until', page)
  }
```
More verbose, but easier to read + maintains.